.github/workflows/test-and-deploy.yml

```diff
@@ -121,6 +97,71 @@ jobs:
         uses: sendgrid/dx-automator/actions/datadog-release-metric@main
         env:
           DD_API_KEY: ${{ secrets.DATADOG_API_KEY }}
+          
+  code-signing:
+    runs-on: windows-latest
+    needs: [ deploy ]
+    steps:
+      - name: Checkout sendgrid-csharp
+        uses: actions/checkout@v2
+
+      - name: Setup .NET Core SDK
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: '3.1.x'
+
+      - name: Set up certificate
+        run: |
+          echo "${{ secrets.SM_CLIENT_CERT_FILE_B64 }}" | base64 --decode > /d/Certificate_pkcs12.p12
+          cat  /d/Certificate_pkcs12.p12
+        shell: bash
+
+      - name: Set variables
+        id: variables
+        run: |
+          dir
+          echo "::set-output name=version::${GITHUB_REF#refs/tags/v}"
+          echo "::set-output name=KEYPAIR_NAME::gt-standard-keypair"
+          echo "::set-output name=CERTIFICATE_NAME::gt-certificate"
+          echo "SM_HOST=${{ secrets.SM_HOST }}" >> "$GITHUB_ENV"
+          echo "SM_API_KEY=${{ secrets.SM_API_KEY }}" >> "$GITHUB_ENV"
+          echo "SM_CLIENT_CERT_FILE=D:\\Certificate_pkcs12.p12" >> "$GITHUB_ENV"
+          echo "SM_CLIENT_CERT_PASSWORD=${{ secrets.SM_CLIENT_CERT_PASSWORD }}" >> "$GITHUB_ENV"
```

Comment:
Will this print "****"  in the logs? If yes, why do we need to print it? If no, lets not print it at all. :)

Reply:
These lines will just set env variables and won't print anything, you can check the run [here](https://github.com/sendgrid/sendgrid-csharp/actions/runs/7814387388/job/21315667765)